ibis/bigquery/tests/conftest.py

```diff
@@ -11,7 +11,12 @@
 
 @pytest.fixture(scope='session')
 def client():
-    return ibis.bigquery.connect(PROJECT_ID, DATASET_ID)
+    ga = pytest.importorskip('google.auth')
```

Comment:
You shouldn't need to use `importorskip` here (you can directly `import google.auth as ga`) since the `importorskip` for `google.cloud.bigquery` in `bigquery/tests/test_client.py` will run before this function is called.

Reply:
note this is already pushed (in #1186). I can fix separately if you'd like.